projects/demo/src/modules/components/rating/examples/1/index.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+
+@Component({
+    selector: 'tui-rating-example-1',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+})
+export class TuiRatingExample1 {
+    rateControl: FormControl = new FormControl(2);
```

Comment:
Trivial type again ) Do you need to merge main, maybe, for the linters to kick in?

Reply:
sorry, fixed